### ISSUE TITLE
Overlay zip redownloads if the setup has stopped after the entire zip has formed

### ISSUE THREAD

**Issue by StamTso** [kubilus1/autoortho#423]
**Describe the bug**
If AutoOrtho region setup is interrupted after the overlay zip has formed in the downloads directory, the next time that the setup begins for the same region AutoOrtho attempts to download the overlay zips again.

**To Reproduce**
1. Start AutoOrtho and begin the setup for a region that has the overlays zip split in many parts (e.g. y_eur_overlays.zip000/001 etc)
2. When the y_eur_overlays.zip has formed in the downloads folder and the parts have been removed, stop the setup and exit the program
3. Start the program again and begin the setup for the same region
4. AutoOrtho starts by downloading the different parts of the overlays and ignores the main file

**Expected behavior**
If the y_eur_overlays.zip exists, it should be able to detect it and skip downloading all the parts again.

**(please complete the following information):**
 - OS: Windows 11 22H2
 - XPlane Version: X-Plane 12.07
 - AutoOrtho Version 0.7.0
 - Filesystem type NTFS

**Additional context**
When I download the zip parts from GitHub and add them to the downloads directory manually, the program skips the download as it should.

**Comment by mercster** [comment]
Yeah, I noticed once you start a download, might as well let it run (I was trying to figure out why download rates were so abysmal.)  There is no recovery from previous partial downloads, it just downloads them again.  I need to look into downloading these with command line because the AutoOrtho tool is having a hell of a time, I think it's being throttled by github, not sure this is the best place to host the multi-gigabyte files...

**Comment by sinistre1973** [comment]
I downloaded Y_NA_Overlay00 to 02 and then the Z_NA_zips manually, then let the AutoOrtho do it's thing again. Then it worked! Tip: use a download manager. The Z_NA zip has 35 parts!

**Comment by mercster** [comment]
> I downloaded Y_NA_Overlay00 to 02 and then the Z_NA_zips manually, then let the AutoOrtho do it's thing again. Then it worked! Tip: use a download manager. The Z_NA zip has 35 parts!

Yeah I ended up letting AutoOrtho download a few zips here and there, and then behind its back wget'ing the zip files and slotting them in place.  I wasn't confident it would "just work" if I fired it up with the zip files already in place, but I noticed it DID recognize the zip files that were already present during a run.  So... haha, kinda brute forced it, but got done in a few hours rather than a few days.

**Comment by StamTso** [comment]
> I think it's being throttled by github, not sure this is the best place to host the multi-gigabyte files...

That is weird because when I download the zips from the scenery repo the speed is significantly faster and I only see throttling when 10 or more files are downloaded in parallel. 

> I downloaded Y_NA_Overlay00 to 02 and then the Z_NA_zips manually, then let the AutoOrtho do it's thing again. 

I did exactly the same for all regions and it was faster and more stable. However, it's not user-friendly at all. The setup tool should be able to:

- check for existing zips and skip downloading if it's not needed. This is done already with the "Z" zips but not with the overlay zips.
- being able to resume a partial download
- being able to retry downloads after a small network timeout. Currently, if there is network instability for a few seconds, the installation is aborted with errors.
